src/Google.Api.Gax.Rest/UserAgentHelper.cs

```diff
@@ -6,8 +6,7 @@
  */
 
```

Comment:
Not sure. It still has code to use with the User-Agent as well... unlike the gRPC one.